src/App.js

```diff
@@ -1,29 +1,11 @@
-import logo from './logo.svg';
-import './App.css';
+import react from 'react';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import Calculator from './components/Calculator';
```

Comment:
- [ ] This import is not working, because you named the Calculator component file `calculator.js`, instead of `Calculator.js`.

Kindly correct this issue so the import works! 💪 

## Explanation

In React, it is customary to name the component files with the first letter Capitalized,  Like `Calculator.js`

## Hints

- Just change the name of the `calculator.js` file to the proper `Calculator.js` and the import will work.

Reply:
thank you very much